tequilapi/endpoints/service.go

```diff
@@ -35,11 +35,17 @@ type serviceRequest struct {
 	// example: 0x0000000000000000000000000000000000000002
 	ProviderID string `json:"providerId"`
 
+	// provider identity passphrase
+	// required: true
+	Passphrase string `json:"passphrase"`
+
 	// service type. Possible values are "openvpn", "wireguard" and "noop"
 	// required: false
 	// default: openvpn
 	// example: openvpn
 	ServiceType string `json:"serviceType"`
+
+	Options json.RawMessage `json:"options"`
```

Comment:
Documentation :(

Reply:
Added description that every service will have own list of options.